python_modules/dagster/dagster/core/definitions/job_definition.py

```diff
@@ -95,6 +99,9 @@ def __init__(
         self._op_selection_data = check.opt_inst_param(
             _op_selection_data, "_op_selection_data", OpSelectionData
         )
+        self._asset_selection_data = check.opt_inst_param(
+            _asset_selection_data, "_asset_selection_data", AssetSelectionData
+        )
```

Comment:
this is internal so we can change it later - but having two sibling args for these SelectionData doesn't feel right, they would never both be set right?

Reply:
yep, I can modify this to only use one arg